tox.ini

```diff
@@ -2,8 +2,14 @@
 envlist = py27, py35, py36, py37, pypy
 
 [testenv]
+# For performance, but also for using "source" with coveragepy (https://github.com/nedbat/coveragepy/issues/268).
+usedevelop = True
 deps =
     pytest
-    pytest-cov
+    # 4.5.2a1 for py37 fix
+    coverage: https://github.com/nedbat/coveragepy/archive/58b210ad8998a9270f4ee6ff0c9054785f579b43.zip
```

Comment:
This is not required anymore when using the "path" already.
The error was due to `paste/` being used, but with tox it would use the package from `.tox/…` anyway.
See https://github.com/nedbat/coveragepy/issues/268.
The proper fix here would be to use a "src" based layout, i.e. move "paste" to "src/paste", but not sure if it is worth it.

Reply:
Given the outcome/review I will adjust this here.

Reply:
So what about this?
I would say to go back to just "coverage" here.
Will push this already.

Reply:
Yeah, go ahead and fix that in whatever way is right.